doc/source/serve/deploy_model_tutorial.rst

```diff
@@ -0,0 +1,328 @@
+====================================
+End-to-End Model Deployment Tutorial
+====================================
+
+By the end of this tutorial you will have learned how to deploy a machine
+learning model locally via Ray Serve.
+
+First, install Ray Serve and all of its dependencies by running the following
+command in your Terminal:
```

Comment:
Is there a broader synonym for Terminal that also applies to Windows? Terminal feels Mac/Linux-specific.

Reply:
I think terminal is fine/well-known. Don't capitalize it though.